crates/teloxide/tests/command.rs

```diff
@@ -331,6 +331,168 @@ fn rename_rules() {
+    #[command(rename_rule = "snake_case")]
+    enum DefaultCommands {
+        /// Start command
+        Start,
+        /// Help command
+        #[command(alias = "h")]
+        Help,
+        #[command(alias = "привет_мир")]
+        HelloWorld(String),
+    }
+
+    assert_eq!(
+        "/start — Start command\n/help, /h — Help command\n/hello_world, /привет_мир",
+        DefaultCommands::descriptions().to_string()
+    );
+}
+
+#[test]
+#[cfg(feature = "macros")]
+fn aliases() {
+    #[derive(BotCommands, Debug, PartialEq)]
+    #[command(rename_rule = "snake_case")]
+    enum DefaultCommands {
+        Start,
+        #[command(aliases = ["h", "помощь"])]
+        Help,
+        #[command(aliases = ["привет_мир"])]
+        HelloWorld(String),
+    }
+
+    assert_eq!(DefaultCommands::Start, DefaultCommands::parse("/start", "").unwrap());
+    assert_eq!(DefaultCommands::Help, DefaultCommands::parse("/help", "").unwrap());
+    assert_eq!(DefaultCommands::Help, DefaultCommands::parse("/h", "").unwrap());
+    assert_eq!(DefaultCommands::Help, DefaultCommands::parse("/помощь", "").unwrap());
+    assert_eq!(
+        DefaultCommands::HelloWorld("username".to_owned()),
+        DefaultCommands::parse("/hello_world username", "").unwrap()
+    );
+    assert_eq!(
+        DefaultCommands::HelloWorld("username".to_owned()),
+        DefaultCommands::parse("/привет_мир username", "").unwrap()
+    );
+}
+
+#[test]
+#[cfg(feature = "macros")]
+fn aliases_help_message() {
+    #[derive(BotCommands, Debug, PartialEq)]
+    #[command(rename_rule = "snake_case")]
+    enum DefaultCommands {
+        /// Start command
+        Start,
+        /// Help command
+        #[command(aliases = ["h", "помощь"])]
+        Help,
+        #[command(aliases = ["привет_мир"])]
+        HelloWorld(String),
+    }
+
+    assert_eq!(
+        "/start — Start command\n/help, /h, /помощь — Help command\n/hello_world, /привет_мир",
+        DefaultCommands::descriptions().to_string()
+    );
+}
+
+#[test]
+#[cfg(feature = "macros")]
+fn hide_aliases_for_unaliases_command() {
+    #[derive(BotCommands, Debug, PartialEq)]
+    #[command(rename_rule = "snake_case")]
+    enum DefaultCommands {
+        /// Start command.
+        Start,
+        /// Show help message.
+        #[command(hide_aliases)]
+        Help,
+    }
+
+    assert_eq!(DefaultCommands::Start, DefaultCommands::parse("/start", "").unwrap());
+    assert_eq!(DefaultCommands::Help, DefaultCommands::parse("/help", "").unwrap());
+
+    assert_eq!(
+        "/start — Start command.\n/help — Show help message.",
+        DefaultCommands::descriptions().to_string()
+    );
+}
+
+#[test]
+#[cfg(feature = "macros")]
+fn hide_aliases_with_alias() {
+    #[derive(BotCommands, Debug, PartialEq)]
+    #[command(rename_rule = "snake_case")]
+    enum DefaultCommands {
+        /// Start.
+        #[command(alias = "s")]
+        Start,
+        /// Help.
+        #[command(alias = "h", hide_aliases)]
```

Comment:
Can you also add a test that shows that `command(alias = "h", hide)` does not show anything?